cmd/rig/cmd/capsule/root/status.go

```diff
@@ -0,0 +1,277 @@
+package root
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/fatih/color"
+	"github.com/rigdev/rig-go-api/api/v1/capsule"
+	"github.com/rigdev/rig-go-api/api/v1/capsule/rollout"
+	"github.com/rigdev/rig-go-api/operator/api/v1/pipeline"
+	"github.com/rigdev/rig/cmd/common"
+	capsule_cmd "github.com/rigdev/rig/cmd/rig/cmd/capsule"
+	"github.com/rigdev/rig/cmd/rig/cmd/flags"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+var boldWhite = color.New(color.FgWhite, color.Bold)
+var red = color.New(color.FgRed)
+
+func (c *Cmd) status(ctx context.Context, cmd *cobra.Command, _ []string) error {
+	statusResp, err := c.Rig.Capsule().GetStatus(ctx, &connect.Request[capsule.GetStatusRequest]{
+		Msg: &capsule.GetStatusRequest{
+			CapsuleId:     capsule_cmd.CapsuleID,
+			ProjectId:     flags.GetProject(c.Scope),
+			EnvironmentId: flags.GetEnvironment(c.Scope),
+		},
+	})
+	if err != nil {
+		return err
+	}
+
+	status := statusResp.Msg.GetStatus()
+	if flags.Flags.OutputType != common.OutputTypePretty {
+		return common.FormatPrint(status, flags.Flags.OutputType)
+	}
+
+	printStatusSummary(status, capsule_cmd.CapsuleID)
+
+	return nil
+}
+func printStatusSummary(s *capsule.Status, capsuleID string) {
+	str := buildCapsuleInfo(s, capsuleID)
```

Comment:
Consider using a `strings.Builder` as multiple concatenations rebuilds the entire string each time